config/crd/bases/anywhere.eks.amazonaws.com_vspheredatacenterconfigs.yaml

```diff
@@ -40,6 +40,8 @@ spec:
             properties:
               datacenter:
                 type: string
+              disableCSI:
```

Comment:
can this be enableCSI with default `true`? that way we don't end up with double negatives in our reasoning "disabled false = true" etc.

Reply:
This was intentional as for older clusters, the default for a boolean is false, so we wouldn't be changing the status quo if this field isn't specified.

Reply:
Resolved in offline conversation. Because this is a value, when we marshal the yaml schema we can't tell if the field is present and set to false or missing and set to false by omission.

Because of this, we're going with the negative field name rather than a positive one.